qemu/tests/openflow_acl_test.py

```diff
@@ -328,6 +328,12 @@ def stop_service(setup_target):
             for script in params.get("copy_scripts").split():
                 script_path = os.path.join(script_dir, script)
                 vm.copy_files_to(script_path, tmp_dir)
+        if params.get("os_type") == "windows":
+            curl_win_path = params.get("curl_win_path", "C:\\curl\\")
+            session.cmd("mkdir %s" % curl_win_path)
```

Comment:
Suggest `session.cmd("dir {0} || mkdir {0}".format(curl_win_path))` to avoid failure when running this in the second time.

Reply:
Will there be a second time? I mean, we used to set 'snapshot=on' and run the test cases, that means after the test, nothing will be added to the original image. So in my understanding, every time we run the case is like the first time.
However, I agree your opinion on check the directory before mkdir. I'll modify the code.

Reply:
Modified. Please help check again, thank you.